modules/signals/src/with-feature.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  SignalStoreFeature,
+  SignalStoreFeatureResult,
+  StateSignals,
+} from './signal-store-models';
+
+/**
+ * Allows passing properties, methods, or signals from a SignalStore
+ * to a feature.
+ *
+ * **Example:**
+ *
+ * ```typescript
+ * signalStore(
+ *   withMethods((store) => ({
+ *     load(id: number): Observable<Entity> {
+ *       return of({ id, name: 'John' });
+ *     },
+ *   })),
+ *   withFeature(
+ *     // 👇 has full access to the store
+ *     (store) => withEntityLoader((id) => firstValueFrom(store.load(id)))
+ *   )
+ * );
+ * ```
+ *
+ * @param featureFactory function returning the actual feature
```

Comment:
```suggestion
 * @description
 * Allows passing properties, methods, or signals from a SignalStore
 * to a feature.
 *
 * @usageNotes
 * ```typescript
 * signalStore(
 *   withMethods((store) => ({
 *     load(id: number): Observable<Entity> {
 *       return of({ id, name: 'John' });
 *     },
 *   })),
 *   withFeature(
 *     // 👇 has full access to the store
 *     (store) => withEntityLoader((id) => firstValueFrom(store.load(id)))
 *   )
 * );
 * ```
 *
 * @param featureFactory function returning the actual feature
```